app/app.go

```diff
@@ -848,6 +763,9 @@ func (app *App) registerUpgradeHandlers() error {
 	})
 
 	app.UpgradeKeeper.SetUpgradeHandler("v2.0.6", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+
+		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+
```

Comment:
Do we need to have this for the v2.0.6 upgrade handler?

Reply:
Ah, we don't need it. I fix it, thanks!

Reply:
thanks!